docs/config_file.md

```diff
@@ -0,0 +1,14 @@
+# Kops config file
+
+Kops will look in the following locations for a config file, in order of priority:
+  - ./
+  - ~/.kops.{file type}
+  - ~/.config/kops/.kops.{file type}
```

Comment:
Can we follow use:

`~/.kops/config`

I would like to follow the convention used by kubectl.